integration_tests/test_basic.py

```diff
@@ -677,9 +677,10 @@ def test_tx_inclusion(cronos, max_gas_wanted):
 
     test against different max-gas-wanted configuration.
     """
+    cmd = f"cronosd start --home . --trace --evm.max-tx-gas-wanted {max_gas_wanted}"
     modify_command_in_supervisor_config(
         cronos.base_dir / "tasks.ini",
-        lambda cmd: f"{cmd} --evm.max-tx-gas-wanted {max_gas_wanted}",
+        lambda _: cmd,
```

Comment:
the change is not fully compatible with before though, just in case the original cmd is not exactly `cronosd start --home .`.

Reply:
seems can only record the origin cmd in global variable to avoid hardcode